data/json/professions.json

```
@@ -5949,5 +5949,22 @@
       "male": [ "briefs" ],
       "female": [ "panties", "bra" ]
     }
-  }
+  },
+  {
+    "type": "profession",
+    "id": "fursuiter",
+    "name": "Fursuiter",
+    "description": "It wasn't long after donning your bulky, technicolor animal costume that things started to go downhill. Now, on top of that, you have the apocalypse to deal with. What a dreadfully VwV situation",
```

```suggestion
    "description": "It wasn't long after donning your bulky, technicolor animal costume that things started to go downhill.  Now, on top of that, you have the apocalypse to deal with.  What a dreadfully VwV situation.",
```
Transifex the thing that's used to localise the game requires 2 spaces between sentences. Also you might want to check if your profession fits any of the scenarios and add them as appropriate in https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/scenarios.json